base/reflection.jl

```diff
@@ -2081,6 +2081,8 @@ function bodyfunction(basemethod::Method)
                     else
                         return nothing
                     end
+                elseif isa(fsym, Core.SSAValue)
+                    fsym = ast.code[fsym.id]
```

Comment:
We might want to add the ability to annotate uses as nothrow, for auto-generated globals we know at lowering time will be defined before they're used.